src/collections/posts/curator-applications-are-now-open.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Curator applications are currently open!
+section: announcements
+category: announcement
+publish: 2024-04-07T18:37:19.983Z
+image: /static/beastsaber-logo-fullsize-square.jpg
+showInPostListing: false
+---
+# Curation Recruitment is Currently Open!
+
+**We are re-opening our curator applications! We’re looking for 3 outstanding applicants of any skill level. These applicants should be people who:**
+
+- have plenty of free time, enthusiasm, and passion for playing and curation,
+
+- are able to demonstrate familiarity with the “Objective Issues” which bar a map from curation as listed in our BeastSaber Curation Team Handbook,
```

Comment:
Seems like it could link the HandBook here quite nicely.

Reply:
I considered this, but it is linked just below, albeit, with different verbiage as "Curation requirements and expectations"... One day we'll update our standard announcement. Guess no harm in linking twice though.